src/Cli/OptionCommand.php

```diff
@@ -0,0 +1,383 @@
+		}
+
+		if ( ! empty( $args ) ) {
+			list( $subcommand ) = $args;
+
+			if ( self::CLI_MANAGED_OPTIONS === $subcommand ) {
+				WP_CLI::line( implode( ', ', self::ALLOWED_OPTIONS ) );
+				WP_CLI::halt( 0 );
+			}
+
+			if ( self::READER_THEMES === $subcommand ) {
+				WP_CLI::line( implode( ', ', wp_list_pluck( $this->reader_themes->get_themes(), 'slug' ) ) );
+				WP_CLI::halt( 0 );
+			}
+
+			/* translators: %s: subcommand */
+			WP_CLI::error( sprintf( __( 'Invalid subcommand: %s', 'amp' ), $subcommand ) );
+		} else {
+			$options = $this->get_options();
+
+			if ( $options instanceof WP_Error ) {
+				/* translators: %s: error message */
+				WP_CLI::error( sprintf( __( 'Could not retrieve options: %s', 'amp' ), $options->get_error_message() ) );
+			}
+
+			WP_CLI\Utils\format_items(
+				'table',
+				array_map(
+					static function ( $option_name, $option_value ) {
+						return compact( 'option_name', 'option_value' );
+					},
+					array_keys( $options ),
+					$options
+				),
+				[ 'option_name', 'option_value' ]
+			);
+		}
+	}
+
+	/**
+	 * Check if the user is set up to use the REST API.
+	 *
+	 * @return true|WP_Error True if the request has permission; WP_Error object otherwise.
+	 */
+	private function check_user_cap() {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			$cap_error = new WP_Error(
+				'amp_rest_cannot_manage_options',
+				__( 'Sorry, you are not allowed to manage options for the AMP plugin for WordPress.', 'amp' )
+			);
+
+			$cap_error->add(
+				'amp_rest_cannot_manage_options_help',
+				__( 'Try using --user=<id|login|email> to set the user context or set it in wp-cli.yml.', 'amp' )
+			);
+
+			return $cap_error;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Get the options.
+	 *
+	 * @return WP_Error|mixed WP_Error on failure, response data on success.
+	 */
+	private function get_options() {
+		$response = $this->do_request( 'GET', self::OPTIONS_ENDPOINT, [] );
+
+		if ( $response->as_error() ) {
+			return $response->as_error();
+		}
+
+		return $response->get_data();
+	}
+
+	/**
+	 * Update an option.
+	 *
+	 * @param string $option_name  Option name.
+	 * @param string $option_value Option value.
+	 *
+	 * @return WP_Error|mixed WP_Error on failure, response data on success.
+	 */
+	private function update_option( $option_name, $option_value ) {
+		$response = $this->do_request(
+			'POST',
+			self::OPTIONS_ENDPOINT,
+			[
+				$option_name => $option_value,
+			]
+		);
+
+		if ( $response->as_error() ) {
+			return $response->as_error();
+		}
+
+		return $response->get_data();
+	}
+
+	/**
+	 * Do a REST Request
+	 *
+	 * @param string $method HTTP method.
+	 * @param string $route REST route.
+	 * @param array  $assoc_args Associative args.
+	 *
+	 * @return \WP_REST_Response Response object.
+	 */
+	private function do_request( $method, $route, $assoc_args ) {
+		$request = new \WP_REST_Request( $method, $route );
```

Comment:
Minor point. The `WP_REST_Request` could be imported above so we don't need to use `\` in this one place:

```suggestion
		$request = new WP_REST_Request( $method, $route );
```